Node/intelligence-ImageCaption/README.md

```diff
@@ -32,8 +33,7 @@ if (hasImageAttachment(session)) {
             .getCaptionFromStream(stream)
             .then(caption => handleSuccessResponse(session, caption))
             .catch(error => handleErrorResponse(session, error));
-    }
-````
+    }````
```

Comment:
Please revert this change, is causing the next line to show as code:
![image](https://user-images.githubusercontent.com/57197/30456632-d27c79f8-997a-11e7-8e67-0aa458aad07f.png)